src/ui/components/NetworkLogo.tsx

```diff
@@ -1,61 +0,0 @@
-import React from 'react';
-import {Image} from 'react-native';
-import PolkadotLog from 'image/Polkadot.png';
```

Comment:
Let's create an issue to check what images we need and clean them up as well. The Polkadot's logo has changed so this is out dated